core/LSU.cpp

```diff
@@ -273,9 +280,10 @@ namespace olympia
         out_cache_lookup_req_.send(mem_access_info_ptr);
     }
 
-    void LSU::getInstFromCache_(const MemoryAccessInfoPtr &memory_access_info_ptr)
-    {
-        auto inst_ptr = memory_access_info_ptr->getInstPtr();
+    // Ready signal from the MMU
+    void LSU::getInstFromCache_(const MemoryAccessInfoPtr &memory_access_info_ptr) {
```

Comment:
This function name is confusing to me.  For a split second I thought I was looking at Fetch!  😆 

The function is not getting any insts from the cache -- it's getting a MMU acknowledgment.  Might as well name it as such..

Reply:
Yeah makes sense. Fixing it now